### ISSUE TITLE
error when trying to pip install

### ISSUE THREAD

**Issue by RobLOlson** [lazeroffmichael/ticktick-py#29]
I receive the following error when I try to pip install this package:
```
Building wheels for collected packages: regex  Building wheel for regex (setup.py) ... error
  error: subprocess-exited-with-error

  × python setup.py bdist_wheel did not run successfully.
  │ exit code: 1
  ╰─> [13 lines of output]
      running bdist_wheel
      running build
      running build_py
      creating build
      creating build\lib.win-amd64-3.10
      creating build\lib.win-amd64-3.10\regex      copying regex_3\__init__.py -> build\lib.win-amd64-3.10\regex
      copying regex_3\regex.py -> build\lib.win-amd64-3.10\regex
      copying regex_3\_regex_core.py -> build\lib.win-amd64-3.10\regex
      copying regex_3\test_regex.py -> build\lib.win-amd64-3.10\regex
      running build_ext
      building 'regex._regex' extension
      error: Microsoft Visual C++ 14.0 or greater is required. Get it with "Microsoft C++ Build Tools": https://visualstudio.microsoft.com/visual-cpp-build-tools/
      [end of output]
```

I realize the fix has something to do with some Microsoft C++ libraries, but I couldn't pin point which ones to download and I have limited drive space.  It was easier for me to just download your package manually and use it locally since the regex dependency never came up.  I'm still not sure where you're using it, but maybe consider removing it?

**Comment by lazeroffmichael** [comment]
Hey everyone. Took a break from this project but will continue with fixes and new features starting next week!

**Comment by V3AI-Seven** [comment]
Yeah, I'm getting the same thing when I try to install, I think we have to download Visual Studio. At least that's what I think.  Not sure though.

**Comment by lazeroffmichael** [comment]
It seems like this is a windows specific issue. @V3AI-Seven are you getting the same issues with the regex dependencies?

**Comment by V3AI-Seven** [comment]
Yeah, it says stuff about "Building Regex Wheel" and then it says something about Regex and failure. It then mentions that I need to install Microsoft Visual C++ 14.

**Comment by V3AI-Seven** [comment]
It also says something about setup.py not being supported.

**Comment by C0rn3j** [comment]
So did you install the required dependency that the error mentions?

https://visualstudio.microsoft.com/visual-cpp-build-tools/

**Comment by r4chl** [comment]
I have Microsoft Visual C++14, and I'm still getting this error.

**Comment by calmwaves111** [comment]
I also encountered this problem
So how to solve it?

Building wheels for collected packages: regex
  Building wheel for regex (pyproject.toml) ... error
  error: subprocess-exited-with-error

  × Building wheel for regex (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [13 lines of output]
      running bdist_wheel
      running build
      running build_py
      creating build
      creating build\lib.win-amd64-cpython-311
      creating build\lib.win-amd64-cpython-311\regex
      copying regex_3\__init__.py -> build\lib.win-amd64-cpython-311\regex
      copying regex_3\regex.py -> build\lib.win-amd64-cpython-311\regex
      copying regex_3\_regex_core.py -> build\lib.win-amd64-cpython-311\regex
      copying regex_3\test_regex.py -> build\lib.win-amd64-cpython-311\regex
      running build_ext
      building 'regex._regex' extension
      error: Microsoft Visual C++ 14.0 or greater is required. Get it with "Microsoft C++ Build Tools": https://visualstudio.microsoft.com/visual-cpp-build-tools/
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for regex
Failed to build regex
ERROR: Could not build wheels for regex, which is required to install pyproject.toml-based projects
